language/de/units.json

```diff
@@ -371,6 +371,7 @@
 			"corcv": "Arbeiterfahrzeug",
 			"cordecom": "Kommandant",
 			"cordemon": "Dämon",
+			"cordeadeye": "Präzisionsschütze" ,
```

Comment:
You only need to add translations in English, the translators will do the rest via Transifex

Reply:
KK, I happen to be German, so I thought I could translate at least my own language.

Should I reverse it?

Reply:
@WatchTheFort What would you say?

Reply:
Better to leave translations to Transifex so they can be reviewed by the translation team

Reply:
Will do.